types/loglinecolor/index.d.ts

```diff
@@ -0,0 +1,71 @@
+// Type definitions for loglinecolor 0.1
+// Project: https://github.com/vixerys/loglinecolor
+// Definitions by: Muunatic <https://github.com/Muunatic>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export class loglinecolor {
+    constructor();
```

Comment:
```suggestion
```
no need for this, it's there by default (implicit)

Reply:
fixed, thanks! @peterblazejewicz